mindsdb/integrations/handlers/scylla_handler/scylla_handler.py

```diff
@@ -240,9 +240,10 @@ def get_columns(self, table_name) -> Response:
         'label': 'Keyspace'
     },
     secure_connect_bundle={
-        'type': ARG_TYPE.STR,
-        'description': 'Path or URL to the secure connect bundle',
-        'required': True,
-        'label': 'Host'
+        'type': ARG_TYPE.PATH,
```

Comment:
We should keep this as string since can accept URLs or S3 files too

Reply:
may be split this field to:
 - secure_connect_bundle: ARG_TYPE.PATH
 - secure_connect_bundle_url: ARG_TYPE.STR
 ?